nyxx_interactions/lib/src/interfaces/ICommandsSync.dart

```diff
@@ -0,0 +1,7 @@
+part of nyxx_interactions;
```

Comment:
File should be moved to dir like `internal/sync`. I don't like structures like here

Reply:
👌🏼